GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -706,6 +710,116 @@ public async Task ExecutePointCollectorAsync(string queueItem, ExecutionContext
             }
         }
 
+        [FunctionName("DiscoverOrganizations")]
+        public Task AutoOnboard([TimerTrigger("0 0 0 * * *") /* execute once per day at midnight*/] TimerInfo timerInfo, ExecutionContext executionContext, ILogger logger)
+        {
+            return this.ExecuteAutoOnboardAsync(executionContext, logger);
+        }
+
+        [FunctionName("DiscoverOrganizations-Dri")]
+        public Task AutoOnboardDri([QueueTrigger("discover-organizations")] string queueItem, ExecutionContext executionContext, ILogger logger)
+        {
+            return this.ExecuteAutoOnboardAsync(executionContext, logger);
+        }
+
+        private async Task ExecuteAutoOnboardAsync(ExecutionContext executionContext, ILogger logger)
+        {
+            DateTime functionStartDate = DateTime.UtcNow;
+            string sessionId = Guid.NewGuid().ToString();
+
+            FunctionContext context = new FunctionContext()
+            {
+                CollectorType = "DiscoverOrganizations",
+                CollectorIdentity = this.configManager.GetCollectorIdentity(),
+                FunctionStartDate = functionStartDate,
+                SessionId = sessionId,
+                InvocationId = executionContext.InvocationId.ToString(),
+            };
+
+            ITelemetryClient telemetryClient = new GitHubApplicationInsightsTelemetryClient(this.telemetryClient, context, logger);
+            try
+            {
+                ICache<RateLimitTableEntity> rateLimiterCache = new AzureTableCache<RateLimitTableEntity>(telemetryClient, "ratelimiter");
+                await rateLimiterCache.InitializeAsync().ConfigureAwait(false);
+                IRateLimiter rateLimiter = new GitHubRateLimiter("*", rateLimiterCache, this.httpClient, telemetryClient, maxUsageBeforeDelayStarts: 80.0, this.apiDomain, throwOnRateLimit: true);
+                ICache<ConditionalRequestTableEntity> requestsCache = new AzureTableCache<ConditionalRequestTableEntity>(telemetryClient, "requests");
```

Comment:
Does calling "Installations" use any usage or subject to rate limiting? It might be, but if it is not, maybe we can permit this to run even during 100% usage (since it has nothing to do)

Reply:
Installations is not subject to rate limiting, I will bump this up to 100. Good catch